games-emulation/mamory/mamory-0.2.25-r1.ebuild

```diff
@@ -40,8 +40,7 @@ src_prepare() {
 
 src_configure() {
 	econf \
-		--includedir=/usr/include \
-		--disable-static
+		--includedir=/usr/include
```

Comment:
Isn't EPREFIX needed? Or does the build system set it internally? (Haven't yet looked)

Reply:
Ahh, forget to check.. I've added it now (couldn't find it in the build system either)